modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaJAXRSSpecServerCodegen.java

```diff
@@ -80,13 +81,6 @@ public JavaJAXRSSpecServerCodegen() {
 
         super.embeddedTemplateDir = templateDir = JAXRS_TEMPLATE_DIRECTORY_NAME + File.separator + "spec";
 
-        for (int i = 0; i < cliOptions.size(); i++) {
-            if (CodegenConstants.LIBRARY.equals(cliOptions.get(i).getOpt())) {
-                cliOptions.remove(i);
-                break;
-            }
-        }
-
```

Comment:
Why these are removed? I don't even recall the reason why we've this block of code.

Reply:
it's not removed but replaced by a generic method (removeOption in line 62) to be used in others methods